src/api/index.test.js

```diff
@@ -273,14 +273,14 @@ it('getCredentials', () => {
   };
   fetchMock.get(/credentials/, data);
   return getCredentials().then(credentials => {
-    expect(credentials).toEqual(data.items);
+    expect(credentials).toEqual(data);
     fetchMock.restore();
   });
 });
 
 it('getCredential', () => {
   const credentialId = 'foo';
-  const data = { fake: 'credential' };
+  const data = { items: 'credential' };
```

Comment:
Is this change actually required? I don't see `items` referenced anywhere in the implementation.

Reply:
don't know why but earlier I was getting an error during testing. I'll see if I can remove this.